gradle/test-launcher.gradle

```diff
@@ -0,0 +1,13 @@
+tasks.withType(Test) {
```

Comment:
Hmm, probably should apply to Android test too.

Reply:
I never really understood the relation between Java and Android. Why does the Java toolchain matter for Android? I thought they are running their own version.

Reply:
Android apps have two types of tests: unit and instrumentation. Unit tests are the same JVM-run tests as any other JAR. Instrumentation is special as in it runs on a device. In the above comment I was referring to these unit tests: https://github.com/mockito/mockito/tree/main/subprojects/androidTest/src/test/java/org/mockitousage/androidtest

Apart from tests Java version matters for Android because they didn't reinvent the wheel (too much), the build still requires .jar/.class inputs, but it spits out .dex files which are essentially over-optimized containers for a transformed bytecode. So it still matters (for AGP) what bytecode target version it's reading (can't polyfill everything, e.g. records), and it matters for Android runtime (on device), because Java cannot be updated on older devices, so we can't call certain methods depending on minSdk.

Reply:
https://github.com/mockito/mockito/pull/3064/commits/2ed1724177e20d8377c707a38e8b9be77aad4b30